Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -1016,9 +1027,29 @@ def search_attributes(demisto_args: dict) -> CommandResults:
     pagination_args_validation(page, limit)
 
     response = PYMISP.search(**args)
+
     if response:
         response_for_context_list = []
         response_for_context_dict = {}
+
+        if 'with_attachments' in args and args.get('with_attachments') == 1:
+            if isinstance(response, list):
+                for attachment in response:
+                    if 'Event' in attachment:
+                        if 'Object' in attachment.get('Event'):
+                            for objects in attachment.get('Event').get('Object'):
+                                if 'Attribute' in objects:
+                                    for object in objects.get('Attribute'):
+                                        if 'value' in object:
+                                            if args.get('value') == object.get('value'):
+                                                for object in objects.get('Attribute'):
```

Comment:
Is there `attribute` key in `attribute` key (something like {"attribute": {"attribute": ["bla", "bla bla"]})?
Because we already running in attribute in line `1041`

Reply:
Thanks @MosheEichler for picking this up. I attached an example output from the response variable. That is basically the base. Due to the structure of the JSON i first iterate through the Attribute under the Object. After I check if the value matches to the argument. If that is the case i check if data exist - if so I run the fileResult command. If you have another idea to make it prettier please let me know.

[response.json](https://github.com/demisto/content/files/12885108/response.json)